iModelCore/BeSQLite/CloudSqlite.cpp

```diff
@@ -271,6 +271,7 @@ CloudResult CloudContainer::Connect(CloudCache& cache) {
 
 /**
  * Disconnect this container from the CloudCache. If the containerDb is open, close it first.
+ * @note This funciton does nothing if there is no cloudcache connected to the cloud container. See CloudContainer::Connect

```

Comment:
this wording is backwards. You connect a CloudContainer to its CloudCache, not the other way around (multiple containers may connect to a single cache.)

Reply:
also, "funciton"